frontend/src/SimulationDisplay.js

```
@@ -0,0 +1,146 @@
+        </BarChart>
+      </ResponsiveContainer>
+
+      <h3 className="text-xl font-bold mt-8 mb-4 text-gray-800">Agent Income Comparison</h3>
+        <ResponsiveContainer width="100%" height={400}>
+            <LineChart>
+                <CartesianGrid strokeDasharray="3 3" />
+                <XAxis dataKey="month" allowDuplicatedCategory={false}/>
+                <YAxis />
+                <Tooltip />
+                <Legend />
+                {Object.keys(dataA.agentChartData.reduce((acc, d) => ({...acc, [d.agent]: true}), {})).map((agentId, i) => (
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)